spec/controllers/v1/people_controller_spec.rb

```diff
@@ -21,7 +21,7 @@
       json_response = JSON.parse(response.body)
 
       expect(json_response).to have_key('error')
-      expect(json_response['error']).to eq("Email can't be blank. Phone can't be blank.")
+      expect(json_response['error']).to eq("person is required")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.